astrodb_utils/fits.py

```diff
@@ -0,0 +1,354 @@
+            ("TDMIN1", "Start in spectral coordinate"),
+            ("TDMAX1", "Stop in spectral coordinate"),
+            ("SPECBAND", "SED.bandpass"),
+            ("APERTURE", "[arcsec] slit width"),
+        ]
+
+    return keywords
+
+
+def make_skycoord(header):
+    # check RA and Dec are in degrees
+    ra_targ = header.get('RA_TARG')
+    if ra_targ is None:
+        ra_targ = header.get('RA')
+        if ra_targ is None:
+            print("RA_TARG or RA is required")
+            ra = None
+            return None
+    else:
+        ra = float(header.get('RA_TARG'))
+        if ra > 360:
+            print("RA_TARG does not appear to be in degrees")
+            print(f"RA_TARG: {ra}")
+            print("RA_TARG should be in degrees")
+            return None
+
+    dec_targ = header.get('DEC_TARG')
+    if dec_targ is None:
+        dec_targ = header.get('DEC')
+        if dec_targ is None:
+            print("DEC_TARG or DEC is required")
+            dec = None
+            return None
+    else:
+        dec = float(header.get('DEC_TARG'))
+
+        if dec > 90:
+            print("DEC_TARG does not appear to be in degrees")
+            print(f"DEC_TARG: {dec}")
+            print("DEC_TARG should be in degrees")
+            return None
+
+    # Check if ra and dec could be read into SkyCoord object and converted to sexagesimal
+    try:
+        coord = SkyCoord(ra,dec, unit=(u.deg, u.deg))
+        coord_str = coord.to_string('hmsdms')
+        print(f"coordinates converted to sexagesimal: {coord_str}")
+        return coord
+    except Exception as e:
+        print(f"coordinates ({ra},{dec}) could not be converted to Skycoord object: {e}")
+        return None
+
+
+def check_simbad_name(header):
+     # search SIMBAD for object name
+    object_name = header.get('OBJECT')
+    simbad_name_results = Simbad.query_object(object_name)
+    coord = make_skycoord(header)
+    
+    if simbad_name_results is None and coord is None:
+        print(f"Object name {object_name} not found in SIMBAD")
+        return False
+    
+    # search for object using RA/Dec coordinates
+    elif simbad_name_results is None and coord is not None:
+        coord_str = coord.to_string('hmsdms')
+        print(f"Object name {object_name} not found in SIMBAD, trying 60\" coord search")
+        simbad_coord_results = Simbad.query_region(coord, radius='0d0m60s')
+        if simbad_coord_results is not None:
+            print(f"SIMBAD objects within 60\" search of {coord_str}:")
+            print(simbad_coord_results)
+            return False
+    
+    if len(simbad_name_results) == 1:
+        print(f"Object name {object_name} found in SIMBAD")
+        print(simbad_name_results)
+        return True
+    else:
+        print(f"Multile object name {object_name} matches found in SIMBAD")
+        print(simbad_name_results)
+        return False
+
+
+def check_ra_dec_simbad(simbad_name_results):
+    result = True
```

Comment:
is this function needed? seems fully commented out?

Reply:
It's a place holder. It's gonna take more time than I have at the moment to write it properly since I want to compare two SkyCoord objects. np.isclose is not the answer here.